src/ugene_plugin_common.pri

```diff
@@ -34,6 +34,7 @@ unix: {
     macx: {
         QMAKE_RPATHDIR += @executable_path/plugins/
     } else {
-        QMAKE_LFLAGS += "-Wl,-rpath,\'\$$ORIGIN/plugins\'"
+        # Make plugins to look for unloaded libraries (Qt*/U2*) in the app folder when loading.
+        QMAKE_LFLAGS += "-Wl,-rpath,\'\$$ORIGIN/..\'"
```

Comment:
Add a path to the current folder.

Reply:
"ugene_plugin_common.pri" includes "ugene_lib_common.pri" that already has the needed line:
        QMAKE_LFLAGS += "-Wl,-rpath,\'\$$ORIGIN\'"

This way all plugins already have a current dir ($ORIGIN) in their library search path.